v2/devices/surya.yml

```diff
@@ -151,3 +159,61 @@ operating_systems:
           - core:user_action:
               action: "recovery"
     slideshow: []
+  - name: "postmarketOS"
+    compatible_installer: ">=0.9.8-beta"
+    options:
+      - var: "variant"
+        name: "Display panel"
+        tooltip: "Display panel manufacturer"
+        link: "https://wiki.postmarketos.org/wiki/Xiaomi_POCO_X3_NFC_(xiaomi-surya)#Know_your_display_panel"
+        type: "select"
+        values:
+          - value: "huaxing"
+            label: "Huaxing"
+          - value: "tianma"
+            label: "Tianma"
```

Comment:
Isn't it a better idea to make this be part of the .json on https://images.postmarketos.org/bpo/index.json? I don't know exactly which changes would be necessary there to distinguish them but it's better than fuzzy matching it in the installer in the end.